frontends/parsers/p4/p4lexer.ll

```diff
@@ -0,0 +1,199 @@
+%{
```

Comment:
Even if it did, the formatting changes would unfortunately make it pretty hard to review. =\